### ISSUE TITLE
How to compile for node 8.10?

### ISSUE THREAD

**Issue by rajesh-vi** [VirgilSecurity/virgil-crypto-javascript#42]
I'm getting following error, how can I re-compile for node 8.10?

```
    module initialization error: Error
    was compiled against a different Node.js version using
    NODE_MODULE_VERSION 64. This version of Node.js requires
    NODE_MODULE_VERSION 57. Please try re-compiling or re-installing
    the module (for instance, using `npm rebuild` or `npm install`).
    at Object.Module._extensions..node (module.js:681:18)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/var/task/node_modules/virgil-crypto/dist/virgil-crypto.cjs.js:5:11)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
```

**Comment by xlwknx** [comment]
@rajesh-vi Hi, try `npm rebuild`

**Comment by rajesh-vi** [comment]
@xlwknx 
I have already tried that command, it doesn't help.

**Comment by vadimavdeev** [comment]
Try running `npm install virgil-crypto` again. There is a post-install script that downloads the native addon compiled for your OS and Node.js version.

**Comment by rajesh-vi** [comment]
Issue is already resolved by reinstalling npm 3.5.2, node v8.10.0 and then install virgil-crypto again.

**Comment by rajesh-vi** [comment]
Closing issue, 
Thanks for the support.
